### PR TITLE
added syncing linear slides and terminology

### PR DESCRIPTION
**Pull request by RuckusRoboticsCode**

### CODE REVIEW

File: src/syncing_two_linear_slide_motors_using_a_pidf_controller.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### Ingredients
+
+1. A PID(F) controller
+2. Tuned PID(F) coefficients
+
+## The Recipe
+
+### The Problem
+
+Linear slides powered by two different motors can end up twisted, with one slide higher than the other. This can happen if the two encoders get out of sync. If you are using the RUN_TO_POSITION motor mode, this causes one slide to be supplied more power than the other and even if you are using a PIDF controlleron each motor, the same problem would occur.
```
Comment (dr-hextanium) [inline src/syncing_two_linear_slide_motors_using_a_pidf_controller.md:12]: fix: typo at "if you are using a PIDF controlleron each motor"

File: src/syncing_two_linear_slide_motors_using_a_pidf_controller.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Methodology
+
+**The BEST way to keep these in sync is to have them be mechanically connected with a bar or a piece of channel. If the two act as one rigid body, then it is a lot less dependent on software. However, we acknowledge that this is not always possible, hence the software solutions.**
```
Comment (dr-hextanium) [inline src/syncing_two_linear_slide_motors_using_a_pidf_controller.md:16]: refactor: "However, this is not always possible" (omission of "we")

File: src/terminology_and_acronyms.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Acronyms
```
Comment (dr-hextanium) [inline src/terminology_and_acronyms.md:1]: love this cute lil section

**Review by dr-hextanium — CHANGES_REQUESTED** [review]

**Comment by RuckusRoboticsCode** [comment]
Okay, I made the edits and shifted code blocks into java files. The indentation is a little weird, but it's fine.

**Pull request merged**